docs/server/source/dev-and-test/setup-run-node.md

```diff
@@ -7,39 +7,61 @@ The BigchainDB core dev team develops BigchainDB on recent Ubuntu and Fedora dis
 
 ## Option A: Using a Local Dev Machine
 
-First, read through the BigchainDB [CONTRIBUTING.md file](https://github.com/bigchaindb/bigchaindb/blob/master/CONTRIBUTING.md). It outlines the steps to setup a machine for developing and testing BigchainDB.
+Read through the BigchainDB [CONTRIBUTING.md file](https://github.com/bigchaindb/bigchaindb/blob/master/CONTRIBUTING.md). It outlines the steps to setup a machine for developing and testing BigchainDB.
 
-Next, create a default BigchainDB config file (in `$HOME/.bigchaindb`):
+### With RethinkDB
+
+Create a default BigchainDB config file (in `$HOME/.bigchaindb`):
 ```text
-bigchaindb -y configure
+$ bigchaindb -y configure rethinkdb
 ```
 
 Note: [The BigchainDB CLI](../server-reference/bigchaindb-cli.html) and the [BigchainDB Configuration Settings](../server-reference/configuration.html) are documented elsewhere. (Click the links.)
 
 Start RethinkDB using:
 ```text
-rethinkdb
+$ rethinkdb
 ```
 
 You can verify that RethinkDB is running by opening the RethinkDB web interface in your web browser. It should be at [http://localhost:8080/](http://localhost:8080/).
 
 To run BigchainDB Server, do:
 ```text
-bigchaindb start
+$ bigchaindb start
 ```
 
-You can [run all the unit tests](running-unit-tests.html) to test your installation.
+### With MongoDB
 
-The BigchainDB [CONTRIBUTING.md file](https://github.com/bigchaindb/bigchaindb/blob/master/CONTRIBUTING.md) has more details about how to contribute.
+Create a default BigchainDB config file (in `$HOME/.bigchaindb`):
+```text
+$ bigchaindb -y configure mongodb
+```
 
+Note: [The BigchainDB CLI](../server-reference/bigchaindb-cli.html) and the [BigchainDB Configuration Settings](../server-reference/configuration.html) are documented elsewhere. (Click the links.)
 
-## Option B: Using a Dev Machine on Cloud9
+Start MongoDB using:
+```text
+$ mongod --replSet=bigchain-rs
+```
 
-Ian Worrall of [Encrypted Labs](http://www.encryptedlabs.com/) wrote a document (PDF) explaining how to set up a BigchainDB (Server) dev machine on Cloud9:
+You can verify that MongoDB is running correctly by checking the output of the
+previous command for the line:
+```text
+waiting for connections on port 27017
+```
+
+To run BigchainDB Server, do:
+```text
+$ bigchaindb start
+```
+
+You can [run all the unit tests](running-unit-tests.html) to test your installation.
```

Comment:
Maybe copy these two sentences ("You can run..." and "The BigchainDB CONTRIBUTING.md file...") to the end of the ```### With RethinkDB``` section as well, otherwise users of RethinkDB might miss them.

An alternative would be to put a new heading before these two sentences, but I have my doubts that readers of the ```### With RethinkDB``` section will notice it.